src/wizards/Wizard.stories.tsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {storiesOf} from "@storybook/react";
+import {State, Store} from "@sambego/storybook-state";
+//import {Modal, ModalSize, ModalBody, ModalFooter} from "./Modal";
+import {Button} from "../forms/button";
+//import {Wizard, WizardStep, WizardSize} from "./Wizard"
+
+import {Wizard, WizardSize} from "./Wizard";
+
+const stepsMedium = [
+    {stepName: "page 1", stepId: 0, stepComponent: <p> Page 1</p>, stepCompleted: false},
+    {stepName: "page 2", stepId: 1, stepComponent: <p> Page 2</p>, stepCompleted: false},
+    {stepName: "page 3", stepId: 2, stepComponent: <p> Page 3</p>, stepCompleted: false},
+];
+
+const stepsLarge = [
+    {stepName: "page 1", stepId: 0, stepComponent: <p> Page 1</p>, stepCompleted: false},
+    {stepName: "page 2", stepId: 1, stepComponent: <p> Page 2</p>, stepCompleted: false},
+    {stepName: "page 3", stepId: 2, stepComponent: <p> Page 3</p>, stepCompleted: false},
+];
+
+const stepsXLarge = [
```

Comment:
The variables "stepsLarge", "stepsMedium" and "stepsXLarge" seems to have similar data, can we create a single data structure which can be used for all the stories?

Reply:
Though the data is same it is belong to different wizard instance. So need to keep them separate.